core/src/main/scala/com/emaginalabs/kuronometer/core/model/BuildExecutionReport.scala

```diff
@@ -0,0 +1,26 @@
+package com.emaginalabs.kuronometer.core.model
+
+case class ProjectInformation(name: Option[String], description: Option[String])
+
+case class ProjectVersion(group: Option[String], version: String)
+
+case class ProjectDefinition(info: ProjectInformation, version: ProjectVersion)
+
+case class Platform(name: String)
+
+case class BuildTool(name: String)
+
+case class StageInfo(name: String, description: Option[String], group: Option[String])
+
+case class StageExecution(executionTimeInNanoseconds: Long, timestamp: Long, skipped: Boolean)
+
+case class Stage(info: StageInfo, execution: StageExecution)
+
+case class BuildStages(stages: List[Stage])
+
+case class BuildExecutionReport(
```

Comment:
I guess you tried any of those http://scalameta.org/scalafmt/#Gradle , but just in case.